### DISCUSSION TITLE
Data extraction from the simulation

### DISCUSSION THREAD

**Discussion by AsthLuucas** [ansys/pyfluent#1870] [Q&A]
I ran the simulation of the test case "mixing_elbow," and everything went well. However, I couldn't extract the desired results. I wanted to extract temperature data at the outlet directly into a Python array, but I could only create a plot of this temperature in the fluent GUI. Could someone help? Here's a picture of the graph creation. I wanted to extract these values as soon as the simulation finishes and store them in an array.

![Capturar](https://github.com/ansys/pyfluent/assets/105306700/000b638a-2d83-4644-ac0e-5ad1bdf5d8dc)

**Comment by AsthLuucas** [comment]
I tried this, but it doesn't return an array as described in the documentation
![Capturar](https://github.com/ansys/pyfluent/assets/105306700/a953a5a5-3b52-4dfe-a396-c3ccd62ce3cd)

**Reply by mkundu1** [reply]
[Here](https://fluent.docs.pyansys.com/version/stable/api/solver/fielddata.html) is the documentation for extracting field-data
